x-pack/test/functional/page_objects/reporting_page.ts

```diff
@@ -145,17 +145,21 @@ export class ReportingPageObject extends FtrService {
     return isToastPresent;
   }
 
-  async setTimepickerInDataRange() {
+  // set the time picker to a range matching 720 documents when using the
+  // reporting/ecommerce archive
+  async setTimepickerInEcommerceDataRange() {
     this.log.debug('Reporting:setTimepickerInDataRange');
-    const fromTime = 'Apr 27, 2019 @ 23:56:51.374';
-    const toTime = 'Aug 23, 2019 @ 16:18:51.821';
+    const fromTime = 'Jun 20, 2019 @ 00:00:00.000';
+    const toTime = 'Jun 25, 2019 @ 00:00:00.000';
```

Comment:
The earlier time range created a search that matched 4675 hits, now it is 720

Reply:
I was only able to reliably reproduce the buggy behaviour with larger CSV exports. I am concerned that if we reduce the number of hits from the search we might hide issues like this in future.

I understand this runs against the changes here that introduced a full snapshot check, but perhaps we could do both tests? One checking a full snapshot (high resolution) and one checking a large export count (with lower resolution snapshot, first 10 and last 10 lines as we had it before).

Let me know what you think!